unity-renderer/Assets/DCLPlugins/ECS7/InternalECSComponents/Interfaces/InternalECSComponentModels.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.ECS7.InternalComponents
+{
+    public class InternalTexturizable
+    {
+        public IList<Renderer> renderers = new List<Renderer>();
```

Comment:
If the collection won't ever hold more than 1 instance of the same renderers then it could be a Hashset instead, that is faster

Reply:
Hashset is faster for looking and removing elements, but painfully slow when iterating through it elements and it instantiate an enumerator every time is iterated.
Since we can expect more elements iterations than element removals I've decided to go for a List which, while iterating, it's as fast and allocation free as an array